tests/e2e/specs/BoundaryEscalationEvent.spec.js

```diff
@@ -0,0 +1,33 @@
+import {
+  dragFromSourceToDest,
+  getElementAtPosition,
+  removeIndentationAndLinebreaks,
+} from '../support/utils';
+import { nodeTypes } from '../support/constants';
+
+describe('Boundary Escalation Event', () => {
+  beforeEach(() => {
```

Comment:
This 'beforeEach()` call is now unnecessary with the refactoring changes that have been made.

Reply:
I saw you just added this back but it shouldn't be necessary. `tests/e2e/support/index.js` has this code in it now and that should be added to every test. Can you check to see if you have those changes in this branch? You should see code from the screenshot below. If you don't have it you may need to rebase from the latest code in develop.

<img width="339" alt="Screen Shot 2019-09-30 at 9 46 55 AM" src="https://user-images.githubusercontent.com/858371/65884849-8c825880-e367-11e9-941d-abbb74ac588c.png">